src/ApiService/ApiService/onefuzzlib/notifications/JinjaTemplateAdapter.cs

```diff
@@ -11,4 +11,351 @@ public class JinjaTemplateAdapter {
+        var jobId = Guid.NewGuid();
+        var taskState = TaskState.Running;
+        var jobState = JobState.Enabled;
+        var os = Os.Linux;
+        var taskType = TaskType.LibfuzzerFuzz;
+        var duration = 100;
+        var project = "some project";
+        var jobName = "job name";
+        var buildName = "build name";
+        var reportContainer = templateRenderContext?.ReportContainer ?? Container.Parse("example-container-name");
+        var reportFileName = templateRenderContext?.ReportFilename ?? "example file name";
+        var reproCmd = templateRenderContext?.ReproCmd ?? "onefuzz command to create a repro";
+        var report = templateRenderContext?.Report ?? new Report(
+                inputUrl.ToString(),
+                null,
+                executable,
+                crashType,
+                crashSite,
+                callStack,
+                callStackSha,
+                inputSha,
+                null,
+                taskId,
+                jobId,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null
+            );
+
+        var task = new Task(
+                jobId,
+                taskId,
+                taskState,
+                os,
+                templateRenderContext?.Task ?? new TaskConfig(
+                    jobId,
+                    null,
+                    new TaskDetails(
+                        taskType,
+                        duration
+                    )
+                )
+            );
+
+        var job = new Job(
+                jobId,
+                jobState,
+                templateRenderContext?.Job ?? new JobConfig(
+                    project,
+                    jobName,
+                    buildName,
+                    duration,
+                    null
+                )
+            );
+
+        var renderer = await NotificationsBase.Renderer.ConstructRenderer(
+            context,
+            reportContainer,
+            reportFileName,
+            report,
+            log,
+            task,
+            job,
+            targetUrl,
+            inputUrl,
+            reportUrl,
+            scribanOnlyOverride: true
+        );
+
+        templateRenderContext ??= new TemplateRenderContext(
+            report,
+            task.Config,
+            job.Config,
+            reportUrl,
+            inputUrl,
+            targetUrl,
+            reportContainer,
+            reportFileName,
+            reproCmd
+        );
+
+        return (renderer, templateRenderContext);
+    }
+
+    public async static Async.Task<(bool didModify, AdoTemplate template)> ConvertToScriban(AdoTemplate template, bool attemptRender = false, IOnefuzzContext? context = null, ILogTracer? log = null) {
```

Comment:
This was copied from `src/ApiService/ApiService/Functions/Migrations/JinjaToScriban.cs`, but I added the option render related args.